python/test/test_repository_standards.py

```diff
@@ -0,0 +1,171 @@
+import unittest
+
+import requests_mock
+
+from python.lib.repository_standards import OrganisationStandardsReport, RepositoryReport
+
+
+class TestOrganisationStandardsReport(unittest.TestCase):
+
+    @staticmethod
+    def mock_repo_data():
+        return {
+            "node": {
+                "hasIssuesEnabled": True,
+                "licenseInfo": None,
+                "description": "repo_description",
+                "isLocked": False,
+                "isDisabled": False,
+                "isArchived": False,
+                "name": "repo_name",
+                "defaultBranchRef": {"name": "main"},
+                "url": "https://github.com",
+                "pushedAt": "2022-01-01",
+                "isPrivate": False,
+                "branchProtectionRules": {"edges": []}
+            }
+        }
+
+    def setUp(self):
+        self.endpoint = 'https://test.com'
+        self.api_key = 'secret'
+        # Hex key for testing
+        self.encryption_key = "5a5263317379674b64564a62516856656a504a6e4b386a4959544236626f44365631516531534a4c3165773d"
+
+        self.mock_report = OrganisationStandardsReport(
+            self.endpoint, self.api_key, self.encryption_key)
+
+    def testSuccessInit(self):
```

Comment:
Should these be snake case? `testSuccessInit`and `testFailInit`

Reply:
Sorry, that's my Go code showing again 😸